WpfApp1/WPFUserInterface.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using FileManager;
+
+namespace WpfApp1;
+
+internal class WPFUserInterface : IUserInterface
+{
+    private MainWindow _mw;
+    public WPFUserInterface(MainWindow mainWindow)
+    {
+            _mw=mainWindow;
+    }
+
+    public void Write(string str)
+    {
+        _mw.LeftTb.Text = _mw.LeftTb.Text + "  " + str;
```

Comment:
Старайтесь не использовать конкатенацию строк. Вместо этого если
- в цикле, то `StringBuilder`
- в остальных случаях - строковая интерполяция
```suggestion
        _mw.LeftTb.Text = $"{_mw.LeftTb.Text} {str}";
```